src/cmd-buildextend-gcp

```diff
@@ -111,7 +111,12 @@ def run_ore():
     checksum = sha256sum_file(build_tarball)
     size = os.path.getsize(build_tarball)
 
-    buildmeta['gcp'] = {'image': f"{base_name}-{args.build.replace('.', '-')}"}
+    gcp_name = f"{base_name}-{args.build.replace('.', '-')}"
+    url = f"https://storage.googleapis.com/{args.bucket}/{base_name}/{args.build}.tar.gz"
```

Comment:
Very minor nit, the `Azure` url is not separately defined but this GCP one is. Might as well make them consistent if you update the commit msg? LGTM either way